src/Extensions/S3Storage/S3StorageEgressProvider.cs

```diff
@@ -43,8 +43,7 @@ public S3StorageEgressProvider(ILogger logger) : base(logger)
             {
                 client = await ClientFactory.CreateAsync(options, artifactSettings, token);
                 uploadId = await client.InitMultiPartUploadAsync(artifactSettings.Metadata, token);
-                int copyBufferSize = options.CopyBufferSize.GetValueOrDefault(0x100000);
```

Comment:
I believe the `0x100000` value was originally a copy-paste from our existing usage in `EgressProvider.cs` - it wasn't actually getting used, because in `MultiPartUploadStream` that would be overridden by the mimimum value anyway.